app/views/devise/user/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
+
```

Comment:
I am not sure if you can leave empty lines between tags.

Reply:
:open_mouth:  I didn't know it was a problem